src/hooks/useStandardFormSchema.ts

```diff
@@ -0,0 +1,13 @@
+import useSWR from 'swr';
+import { openmrsFetch } from '@openmrs/esm-framework';
+
+export function useStandardFormSchema() {
+  const url = 'https://json.openmrs.org/form.schema.json';
+  const { data, error, isLoading } = useSWR<{ data }, Error>(url, openmrsFetch);
```

Comment:
Prefer [useSWRImmutable](https://swr.vercel.app/docs/revalidation#disable-automatic-revalidations) here as the schema is unlikely to be a resource that changes often.